### PR TITLE
condition cleanup

### PR DESCRIPTION
**Pull request by amazingalek**

### CODE REVIEW

File: QSB/Animation/AnimationType.cs [hunk 1]
```diff
@@ -2,12 +2,12 @@
...
-		PlayerSuited,
-		PlayerUnsuited,
-		Riebeck
+		Chert = 0,
+		Esker = 1,
+		Feldspar = 2,
+		Gabbro = 3,
+		PlayerSuited = 4,
+		PlayerUnsuited = 5,
+		Riebeck = 6
```
Comment (misternebula) [inline QSB/Animation/AnimationType.cs:11]: This is done automatically though? no need to write unneeded stuff

File: QSB/Animation/AnimatorMirror.cs [hunk 2]
```diff
@@ -35,7 +35,7 @@ public void Init(Animator from, Animator to)
 			}
 		}
 
-		private void Update()
+		public void Update()
```
Comment (misternebula) [inline QSB/Animation/AnimatorMirror.cs:38]: why? 😛

File: QSB/Instruments/InstrumentsManager.cs [hunk 3]
```diff
@@ -10,28 +10,28 @@ namespace QSB.Instruments
...
 		{
-			rootObj = root;
+			_rootObj = root;
 			gameObject.AddComponent<CameraManager>();
 
 			QSBInputManager.ChertTaunt += () => StartInstrument(AnimationType.Chert);
 			QSBInputManager.EskerTaunt += () => StartInstrument(AnimationType.Esker);
 			QSBInputManager.FeldsparTaunt += () => StartInstrument(AnimationType.Feldspar);
 			QSBInputManager.GabbroTaunt += () => StartInstrument(AnimationType.Gabbro);
 			QSBInputManager.RiebeckTaunt += () => StartInstrument(AnimationType.Riebeck);
```
Comment (misternebula) [inline QSB/Instruments/InstrumentsManager.cs:26]: cant't these also be the OnChertTaunt, OnEskerTaunt etc. ?

File: QSB/TimeSync/TimeSyncUI.cs [hunk 4]
```diff
@@ -75,7 +77,7 @@ private void OnWillRenderCanvases()
 			switch (_currentType)
 			{
 				case TimeSyncType.Fastforwarding:
-					text = $"{minutes.ToString("D2")}:{seconds.ToString("D2")}"
+					text = $"{minutes:D2}:{seconds:D2}"
```
Comment (misternebula) [inline QSB/TimeSync/TimeSyncUI.cs:80]: 😮

File: QSB/Animation/AnimationType.cs (hunk 1 above)
Comment (amazingalek) [reply QSB/Animation/AnimationType.cs:11]: it's not needed, you're right. But, maybe in the future, someone builds on top of QSB, and if you reorder enum values without explicit number values, it'll change stuff for them. Feel free to remove the numbering :)

**Pull request merged**